solutions/ac-MultipleOf2019.mdx

```diff
@@ -2,11 +2,53 @@
 id: ac-MultipleOf2019
 source: AtCoder Beginner Contest
 title: Multiple of 2019
-author: Maggie Liu, Brad Ma, Mohammad Nour Massri
+author: Maggie Liu, Brad Ma, Mohammad Nour Massri, David Zhang
 ---
 
 [Official Analysis](https://img.atcoder.jp/abc164/editorial.pdf#page=10)
 
+## Explanation
+In order to store sections of the string, we can make a suffix array $S$, such that:
+
+$${S_i = s_n + 10*s_{n-1} +100*s_{n-2} + ... + {10^{n-i}} * s_{i}}$$
+
+For example, if our string was $$1234$$ and the modulus was $$3$$, the suffix array would be
```

Comment:
> modulus was $$3$$

? i thought the mod was 2019?

Reply:
earlier, in the example, I used 3 instead of 2019

Reply:
yeah, just keep it to 2019, it's less confusing

Reply:
```suggestion
For example, if our string was $$1234$$ and the modulus was $$3$$ (instead of $$2019$$), the suffix array would be
```
just for clarification